notebooks/climatiq.ipynb

```diff
@@ -505,7 +377,7 @@
     "If your data comes with an industry specification code Climatiq API allows you to query emission factors, based on their NACE or ISIC industry classification codes. At the moment this endpoint works only with spend-based datasets, in particular EXIOBASE and GHG Protocol. \n",
```

Comment:
Suggest to edit in this way: If your data comes with an industry specification code Climatiq API allows you to query emission factors, based on their NACE, MCC or UNSPSC industry classification codes. At the moment this endpoint works only with spend-based dataset- EXIOBASE.


The ISIC codes are broken right, maybe we should remove them for now?